templates/common/_base/units/ovs-configuration.service.yaml

```diff
@@ -9,7 +9,8 @@ contents: |
   Requires=openvswitch.service
   Wants=NetworkManager-wait-online.service
   After=NetworkManager-wait-online.service openvswitch.service network.service nodeip-configuration.service
-  Before=network-online.target kubelet.service crio.service node-valid-hostname.service
```

Comment:
So severing the ordering here of `Before=node-valid-hostname.service` isn't something I felt comfortable with personally.

It may actually be safe and make sense to be clear - I just don't know myself.

And that's why I was trying to do a more wholesale move of all this stuff to be `After=network-online.target`, but still maintaining the order between them.

If this *is* a safe move to make, then indeed your PR here is a much smaller and then less risky change to make than mine.